portal-docs/observability.md

```diff
@@ -50,11 +53,11 @@ We will discuss each of them below.
 
 You will see the options bar at the top of the observability page. There, by changing the values provided by the dropdown menus, certain configurations could be changed as follows.
 
-1- Change the resource, remote function or function you need to observe
-2- Change the time window of throughput and latency graphs  
-3- Change the timezone of throughput, latency graphs, and the logs view
-4- Change the refresh interval  
-5- Turn sharing on/off  
+1. Change the resource, remote function or function you need to observe
+2. Change the time window of throughput and latency graphs  
+3. Change the timezone of throughput, latency graphs, and the logs view
+4. Change the refresh interval  
+5. Turn sharing on/off  
 
```

Comment:
Since this is also a list of possible configuration changes that can be done, better to use bullet points instead of numbering. Let's use numbering only in places where following an order is necessary.